h2/src/main/org/h2/expression/function/Function.java

```diff
@@ -1534,6 +1540,42 @@ protected Value getValueWithArgs(Session session, Expression[] args) {
         return result;
     }
 
+    private Value regexpSubstr(Value inputString, Value regexpArg, Value positionArg, 
+            Value occurrenceArg, Value regexpModeArg, Value subexpressoinArg) {
+        String regexp = regexpArg.getString();
+
+        if (regexpArg == ValueNull.INSTANCE || positionArg == ValueNull.INSTANCE 
+            || occurrenceArg == ValueNull.INSTANCE || subexpressoinArg == ValueNull.INSTANCE) {
+            return ValueNull.INSTANCE;
+        }
+
+        int position = positionArg != null ? positionArg.getInt() - 1 : 0;
+        int requestedOccurrence = occurrenceArg != null ? occurrenceArg.getInt() : 1;
+        String regexpMode = regexpModeArg != null ? regexpModeArg.getString() : null;
+        int subexpression = subexpressoinArg != null && subexpressoinArg != ValueNull.INSTANCE ? subexpressoinArg.getInt() : 0;
```

Comment:
`subexpressoinArg` can't be `ValueNull.INSTANCE` here, and this parameter has a typo in its name, should be `subexpressionArg`.

(`regexpModeArg` can, but `ValueNull.INSTANCE.getString()` simply returns `null`.)

Reply:
33b1736